packages/convai-widget-core/src/types/config.ts

```diff
@@ -40,12 +60,34 @@ export type AvatarConfig =
     };
 
 export const DefaultTextContents = {
-  start_call_text: "Start a call",
-  end_call_text: "End",
-  action_text: "Need help?",
-  expand_text: "Chat with AI",
-  listening_text: "Listening",
-  speaking_text: "Talk to interrupt",
+  main_label: "Need help?",
+  start_call: "Start a call",
+  new_call: "New call",
+  end_call: "End",
+  mute_microphone: "Mute microphone",
+  change_language: "Change language",
+  collapse: "Collapse",
+  expand: "Expand",
+  copied: "Copied!",
+  accept_terms: "Accept",
+  dismiss_terms: "Cancel",
+
+  listening_status: "Listening",
+  speaking_status: "Talk to interrupt",
+  connecting_status: "Connecting",
+
+  input_label: "Text message input",
+  input_placeholder: "Or send a message",
```

Comment:
I find the "Or" as the start of the placeholder to look weird in the UI. Did we consider some other options?